src/shared/lib/StoreUtils.ts

```diff
@@ -263,7 +263,6 @@ export function fetchCopyNumberSegmentsForSamples(samples: Sample[],
                 sampleId: sample.sampleId,
                 studyId: sample.studyId
             })),
-            chromosome,
```

Comment:
This was throwing an error during the build.. checked `CBioPortalAPI.ts` and chromosome doesn't seem to be a param. Please correct me if I'm wrong.

Reply:
try making this pull request against master (and rebase on master)

Reply:
@alisman thanks, will do